packages/vega-typings/types/runtime/index.d.ts

```diff
@@ -20,6 +20,17 @@ export interface Loader {
   file: (filename: string) => Promise<string>;
 }
 
+export interface ToCanvasOptionsWithoutContext {
+  type?: string;
+  context?: any;
+}
+
+export interface ToCanvasOptionsWithContext {
+  intoContext?: any;
```

Comment:
This should be 'externalContext' now right?

Reply:
Thanks, good catch! I've pushed an update.